crates/ruma-common/src/events/kinds.rs

```diff
@@ -300,7 +300,7 @@ pub struct StrippedStateEvent<C: PossiblyRedactedStateEventContent> {
 }
 
 /// A minimal state event, used for creating a new room.
-#[derive(Clone, Debug, Event)]
+#[derive(Clone, Debug, Event, Serialize)]
```

Comment:
This wouldn't work, we need to add the `type` field during serialization.

Much like the manual implementation of `ToDeviceEvent` in the same file.

Reply:
done